lefthook.yml

```diff
@@ -15,19 +15,15 @@ pre-commit:
       glob: "*.{rb,rake}"
       run: bundle exec stree check Gemfile {staged_files}
     prettier:
-      glob: "*.js"
-      include: "app/assets/javascripts|plugins/.+?/assets/javascripts"
-      run: pnpm pprettier --list-different {staged_files}
-    prettier-scss:
-      glob: "*.scss"
-      include: "app/assets/stylesheets|plugins/.+?/assets/stylesheets"
+      glob: "*.{js,gjs,hbs,scss,css}"
```

Comment:
prettier and eslint should also include `cjs` and `mjs` files. (those aren't completely covered in our npm scripts, that should be fixed)